sdk/compute/Azure.ResourceManager.Compute/src/Customize/VirtualMachineCollection.cs

```diff
@@ -28,5 +30,13 @@ public virtual Pageable<VirtualMachineResource> GetAll(string filter, Cancellati
             HttpMessage NextPageRequest(int? pageSizeHint, string nextLink) => _virtualMachineRestClient.CreateListNextPageRequest(nextLink, Id.SubscriptionId, Id.ResourceGroupName, filter, null);
             return PageableHelpers.CreatePageable(FirstPageRequest, NextPageRequest, e => new VirtualMachineResource(Client, VirtualMachineData.DeserializeVirtualMachineData(e)), _virtualMachineClientDiagnostics, Pipeline, "VirtualMachineCollection.GetAll", "value", "nextLink", cancellationToken);
         }
```

Comment:
could we also change the implementation of two above existing methods? They should call their corresponding public methods as well. We might miss that in our last review